Modelica/Electrical/Machines.mo

```diff
@@ -1168,9 +1168,9 @@ and accelerating inertias.<br>At time tStep a load step is applied.</p>
           annotation (Placement(transformation(
               extent={{-10,10},{10,-10}},
               rotation=270)));
-        Blocks.Sources.CombiTimeTable
-                                 dutyCycle(table=[0,0; 1,1; 4,1; 5,0; 10,0; 11,
-              -1; 14,-1; 15,0; 20,0], extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic)
+        Blocks.Sources.CombiTimeTable dutyCycle(
```

Comment:
To me the change for dutyCycle just looks like formatting change; am I missing something?

Reply:
Yes, cosmetics only. SimulationX has some issue when displaying arrays with line breaks:

![grafik](https://user-images.githubusercontent.com/14896695/64696889-ddf1a300-d49f-11e9-920a-116f5f8eef61.png)